JS/night.js

```diff
@@ -0,0 +1,30 @@
+// // NIGHT MODE
+
+let nightButton = document.querySelector('.nightMode')
+
+let nightMain = document.querySelector('.main-grid')
+let nightBody = document.querySelector('.body')
+let nightAbout = document.querySelector('.about')
+let nightHeadline = document.querySelector('.headline')
+let nightHeadlineTag = document.querySelector('.headline-tag')
+let nightButtonLogout = document.querySelector('.button-logout')
+let nightFooter = document.querySelector('.footer-grid')
+let nightIcon = document.querySelector('.button')
+let nightMode = document.querySelector('.nightMode')
+
+nightButton.addEventListener('click', event => {
+  nightMain.classList.toggle('main-grid-dark')
+  nightBody.classList.toggle('body-dark')
+  nightAbout.classList.toggle('about-dark')
+  nightHeadline.classList.toggle('headline-dark')
+  nightHeadlineTag.classList.toggle('headline-tag-dark')
+  nightButtonLogout.classList.toggle('button-logout-dark')
+  nightFooter.classList.toggle('footer-grid-dark')
+  nightIcon.classList.toggle('button-night')
+  nightMode.classList.toggle('nightModeDark')
+})
+
+nightButton.addEventListener('click', event => {
+  event.target.textContent =
+    event.target.textContent == 'Night Mode' ? 'Night Mode OFF' : 'Night Mode'
```

Comment:
Please use `===` instead of `==`
`===` checks both value and type equality, and is less error prone